<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupWidget</class>
 <widget class="QWidget" name="GroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="group_label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gruppe:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{
border: 1.5px solid rgb(190, 190, 190);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="driver1">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Fahrer 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="name1">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	color: rgb(255, 0, 0)
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Platz:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="rank_combo1">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>1.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4.</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="driver2">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Fahrer 2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="name2">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	color: rgb(255, 0, 0)
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Platz:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="rank_combo2">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>1.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4.</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="driver3">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Fahrer 3:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="name3">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	color: rgb(255, 0, 0)
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Platz:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="rank_combo3">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>1.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4.</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="driver4">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Fahrer 4:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="name4">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	color: rgb(255, 0, 0)
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Platz:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="rank_combo4">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>1.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4.</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
